src/FSM/spec/State/StateTransitionBuilderSpec.php

```diff
@@ -36,8 +36,12 @@ function it_throws_exception_when_label_is_missing(State $to)
     function it_builds_default_transition_based_on_given_details(State $from)
     {
         $to     = new DefaultState('S1');
-        $guard  = function() { return true; };
```

Comment:
Code style fixes. 
Not sure how I managed to apply this to tests. I blame PHPStorm, but can roll it back if needed.